rpcs3/rpcs3qt/core_tab.cpp

```diff
@@ -126,7 +125,7 @@ core_tab::core_tab(std::shared_ptr<emu_settings> settings, QWidget *parent) : QW
 	for (const auto& prxf : fs::dir(lle_dir))
 	{
 		// List found unselected modules
-		if (prxf.is_directory || (prxf.name.substr(std::max(size_t(3), prxf.name.length()) - 3)) != "prx")
+		if (prxf.is_directory || (prxf.name.substr(max(size_t(3), prxf.name.length()) - 3)) != "prx")
```

Comment:
This line appears to be what broke travis/appveyor. Why was it changed from std::max?